inference-engine/src/transformations/src/transformations/op_conversions/convert_mvn1_to_mvn6.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/op_conversions/convert_mvn1_to_mvn6.hpp"
+
+#include <ngraph/rt_info.hpp>
+
+#include <numeric>
+
+#include <ngraph/opsets/opset2.hpp>
+#include <ngraph/opsets/opset6.hpp>
+#include <ngraph/pattern/op/wrap_type.hpp>
+
+NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvertMVN1ToMVN6, "ConvertMVN1ToMVN6", 0);
+
+ngraph::pass::ConvertMVN1ToMVN6::ConvertMVN1ToMVN6() {
+    auto mvn = pattern::wrap_type<ngraph::op::v0::MVN>();
+
+    ngraph::matcher_pass_callback callback = [](pattern::Matcher& m) {
+        auto match_root = m.get_match_root();
+        auto mvn_node = std::dynamic_pointer_cast<ngraph::op::v0::MVN>(match_root);
```

Comment:
Remove `match_root` as it is useless.
```suggestion
        auto mvn_node = std::dynamic_pointer_cast<ngraph::opset1::MVN>(m.get_match_root());
```